File: core/src/galenscovell/hinterstar/ui/components/gamescreen/hud/InfoPanel.scala
package galenscovell.hinterstar.ui.components.gamescreen.hud

import com.badlogic.gdx.scenes.scene2d.ui.{Label, Table}
import galenscovell.hinterstar.util.Resources


class InfoPanel(message: String) extends Table {
  this.setFillParent(true)

  val pauseTable: Table = new Table
  pauseTable.setBackground(Resources.blueButtonNp0)
  val pauseLabel: Label = new Label(message, Resources.labelLargeStyle)

  pauseTable.add(pauseLabel).pad(6)

  this.add(pauseTable).expand.height(32).center.top.padTop(80)
}

File: core/src/galenscovell/hinterstar/ui/components/gamescreen/hud/InfoPanel.kt
package galenscovell.hinterstar.ui.components.gamescreen.hud

import com.badlogic.gdx.scenes.scene2d.ui.*
import galenscovell.hinterstar.util.Resources


class InfoPanel(message: String) : Table() {

    init {
        this.setFillParent(true)

        val pauseTable: Table = Table()
        pauseTable.background = Resources.blueButtonNp0
        val pauseLabel: Label = Label(message, Resources.labelMediumStyle)

        pauseTable.add(pauseLabel).pad(6f)

        this.add(pauseTable).expand().height(32f).center().top().padTop(80f)
    }
}
